### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by mybits**
PT: https://www.pivotaltracker.com/story/show/107913848
design: https://drive.google.com/drive/folders/0B_zI_K-QvNpddHYwY2pOdFVkY1k

### CODE REVIEW

**Comment by tomsabin** [comment]
Did we decide to not include the company registration number in the end?

File: source/assets/stylesheets/styles/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+.footer-container__address-full {
+  display: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    display: block;
+  }
+}
+
+.footer-container__registration-address {
```
Comment (tomsabin) [inline source/assets/stylesheets/styles/_footer.scss:85]: I'm guessing that this should be displayed somewhere, including the SA registration details..?

**Comment by tomsabin** [comment]
The button and input form is rendering incorrectly (rounded borders) on iOS:

slack for ios upload png

**Comment by tomsabin** [comment]
The text input functionality is a bit odd - the placeholder padding is ~30px from the left but when you focus on it, there is no padding.

screen shot 2015-11-20 at 14 53 16
screen shot 2015-11-20 at 14 52 43
screen shot 2015-11-20 at 14 53 06

File: source/_footer.erb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      </a>
+    <div class="footer-container__address-full">
+      <p>303 Buchanan Chambers,</p>
+      <p>Cnr Pearce & Warwick Street</p>
+      <p>Claremont, 7735</p>
+      <p>South Africa</p>
+     </div>
+    </div>
+    <div class="footer-container__newsletter">
+       <p>Sign up to unboxed updates</p>
```
Comment (tomsabin) [inline source/_footer.erb]: Capital U? I wonder whether we are "unboxed" or "Unboxed"... I would have thought the latter, but only the logo is lowercase.

File: source/_footer.erb [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        <a class="social-icons__icon-container" href="http://twitter.com/Ubxd" target="_blank">
+          <div class="social-icons__icon--twitter">
+            <%= image_tag 'twitter.png', class: 'social-icons__image', alt: 'Twitter' %>
+          </div>
+        </a>
+        <a class="social-icons__icon-container" href="http://www.linkedin.com/company/unboxed-consulting" target="_blank">
+          <div class="social-icons__icon--linkedin">
+            <%= image_tag 'linkedin.png', class: 'social-icons__image', alt: 'LinkedIn' %>
+          </div>
+        </a>
```
Comment (tomsabin) [inline source/_footer.erb]: No more Facebook? :open_mouth:

**Comment by tomsabin** [comment]
screen shot 2015-11-20 at 15 23 17

I'm trying to figure out why the paragraph tag within `footer-container__newsletter` has such a large height. While it might render correctly, I can't help but think something will go wrong here because of it.

It is probably the three floated tiles above, therefore this element might require a clearfix.

File: source/assets/stylesheets/styles/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  @include breakpoint($tablet-breakpoint) {
+    display: none;
+  }
+}
+
+.footer-container__newsletter {
+  font-weight: 100;
+
+  @include breakpoint($tablet-breakpoint) {
+    padding-left: 16px;
```
Comment (tomsabin) [inline source/assets/stylesheets/styles/_footer.scss]: Two things here:
1. I think we have quite a mixture of padding dimensions, but we should try and be consistent with them - I've been trying to use 20px left and right as a minimum after realising this (I have also update the project story styles on the `homepage` branch). Currently, on those pages this footer fits nicely with the related content above (which is also 16px), but on other pages such as the blog, the padding is 20px.
2. Secondly is because we only have a left padding, declaring 50% for the newsletter is showing larger than 50% (maybe 51/52%) because there is no padding on the right. Because of that the newsletter looks ever-so-slightly off. Hopefully the screenshot will be able to explain.

screen shot 2015-11-20 at 15 28 33

File: source/assets/stylesheets/styles/_footer_social_icons.scss [hunk 5]
```diff
@@ -1,5 +1,4 @@
 .footer-social-icons {
-  background-color: $footer-social-icons__background-color;
   text-align: right;
 }
```
Comment (tomsabin) [inline source/assets/stylesheets/styles/_footer_social_icons.scss]: Is this file needed anymore?

File: source/_footer.erb [hunk 6]
```diff
@@ -0,0 +1,67 @@
+<div class="footer">
+  <div class="footer-container">
+    <div class="footer-container__address-tile">
+      <p class="footer-container__company-name">
+        unboxed
+      </p>
+      <p>hello@unboxed.co</p>
```
Comment (tomsabin) [inline source/_footer.erb]: Could this be a `mailto`?

File: source/_footer.erb [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+        </a>
+        <a class="social-icons__icon-container" href="http://www.linkedin.com/company/unboxed-consulting" target="_blank">
+          <div class="social-icons__icon--linkedin">
+            <%= image_tag 'linkedin.png', class: 'social-icons__image', alt: 'LinkedIn' %>
+          </div>
+        </a>
+      </div>
+      <div class="footer-container__registration-address">
+        <p>registered in England and Wales</p>
+        <p>05440833</p>
```
Comment (tomsabin) [inline source/_footer.erb]: _Maybe_ we could link to https://beta.companieshouse.gov.uk/company/05440833 :stuck_out_tongue:

File: source/_footer.erb [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+      </div>
+    </div>
+    <div class="footer-container__address-tile">
+      <p class="footer-container__address-city"><a href="http://goo.gl/maps/AuKrs" target="_blank"><i class="fa fa-map-marker footer__map-icon"></i>
+        London
+      <span>office</span></a></p>
+      <a class="footer-container__address-phone" href="tel:+442071834250"><i class="fa fa-phone footer__phone-icon"></i>
```
Comment (tomsabin) [inline source/_footer.erb:30]: :+1:

File: source/_footer.erb [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+        +44 20 7183 4250
+      </a>
+      <div class="footer-container__address-full">
+        <p>17 Blossom Street</p>
+        <p>London, E1 6PL</p>
+        <p>United Kingdom</p>
+      </div>
```
Comment (tomsabin) [inline source/_footer.erb:36]: Similar to the mobile view, we could link the address to Google maps

Comment (mybits) [inline source/_footer.erb:36]: We got it in London/Cape Town office - guess that would be enough?

File: source/_footer.erb (hunk 6 above)
Comment (mybits) [inline source/_footer.erb]: Done!

File: source/assets/stylesheets/styles/_footer_social_icons.scss (hunk 5 above)
Comment (mybits) [inline source/assets/stylesheets/styles/_footer_social_icons.scss]: Good spot! Removed

File: source/_footer.erb (hunk 7 above)
Comment (mybits) [inline source/_footer.erb]: I'll ask Graeme about it :)

File: source/assets/stylesheets/styles/_footer.scss (hunk 4 above)
Comment (mybits) [inline source/assets/stylesheets/styles/_footer.scss]: Oh yes, that was actually changed in later commits, but now since we've changed the layout I guess it'll need another review :)

File: source/_footer.erb (hunk 2 above)
Comment (mybits) [inline source/_footer.erb]: Yes, agreed! Changed it to capital 'U'

File: source/_footer.erb (hunk 7 above)
Comment (mybits) [inline source/_footer.erb]: He said, not necessary :) so I'll leave it as it is

File: source/_footer.erb [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+          London
+        <span>office</span></a></p>
+        <a class="footer-container__address-phone" href="tel:+442071834250"><i class="fa fa-phone footer__phone-icon"></i>
+          +44 20 7183 4250
+        </a>
+        <div class="footer-container__address-full">
+          <p>17 Blossom Street</p>
+          <p>London, E1 6PL</p>
+          <p>United Kingdom</p>
+        </div>
```
Comment (tomsabin) [inline source/_footer.erb]: It has just occurred to me that we could probably use an [`address`](http://www.w3schools.com/tags/tag_address.asp) tag here.

> The address element provides contact information for a document or part of a document. Information provided by address may include the names of the document’s maintainers, links to the maintainers’ Web pages, e-mail addresses for feedback, postal addresses, phone numbers, and so on. The address element is not appropriate for all postal and e-mail addresses; it should be reserved for providing such information about the contact people for the document.

**Comment by mybits** [comment]
@tomsabin thanks for all the comments, all have been addressed in separate commits

**Comment by tomsabin** [comment]
LGTM :v:
